Core/include/Acts/EventData/MultiTrajectory.hpp

```diff
@@ -153,30 +166,41 @@ class TrackStateProxy {
   constexpr static auto ProjectorFlags = Eigen::RowMajor | Eigen::AutoAlign;
   using Projector =
       Eigen::Matrix<typename Covariance::Scalar, M, N, ProjectorFlags>;
-  using EffectiveProjector =
-      Eigen::Matrix<typename Projector::Scalar, Eigen::Dynamic, Eigen::Dynamic,
-                    ProjectorFlags, M, N>;
+  using EffectiveProjector = Eigen::Matrix<typename Projector::Scalar,
+                                           Eigen::Dynamic,
+                                           Eigen::Dynamic,
+                                           ProjectorFlags,
+                                           M,
+                                           N>;
```

Comment:
Disclaimer: taste discussion (I'll accept this when overruled), but I find this repulsive.

Reply:
Definitely taste. From this side, this looks immensely more readable.